pytorch_lightning/metrics/classification/stat_scores.py

```diff
@@ -0,0 +1,261 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Optional, Any, Callable
+
+import torch
+from pytorch_lightning.metrics import Metric
+from pytorch_lightning.metrics.functional.stat_scores import _stat_scores_update, _stat_scores_compute
+
+
+class StatScores(Metric):
+    """Computes the number of true positives, false positives, true negatives, false negatives.
```

Comment:
maybe it would be helpfull linking to this wiki page: https://en.wikipedia.org/wiki/Type_I_and_type_II_errors

Reply:
I'll add this one also, as it has the nice table https://en.wikipedia.org/wiki/Confusion_matrix#Table_of_confusion